preview-site-src/elements/admonition.adoc

```diff
@@ -1,7 +1,7 @@
 = Admonition
 :keywords: admonition, admonitions, warning, section, notes
 :page-component-name: elements
-:page-notice-banner-message: This is a <b>custom</b> notice message with a <a href="https://salesforce.com" target="_blank" rel="noopener">link</a>.
+:page-support-status: eolScheduledFeature
```

Comment:
Not sure what this is for - is it just for preview? (I'm not sure why Gary had the original.)

Reply:
Yep, this is just to preview it locally. Thanks for the quick turnaround!